### PR TITLE
[13.0] [MIG] Partner Aging

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Interactive Partner Aging at any date
=====================================

This module creates new customer/supplier aging as of a given date.

The default Aged Partner balance report is related to a specific date and a
static PDF that is based on the difference between credits and debits, not
based on documents such as Invoices and Payments.
This module provides an interactive view with details of the invoice.
This module does not consider unapplied credits.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@max3903 @smangukiya Can you please review it?

File: partner_aging/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2012 Open Source Integrators
```
Comment (smangukiya) [inline partner_aging/__manifest__.py]: Can you please change the Copyright from 2012 to 2019 on all the files?

**Review by smangukiya — CHANGES_REQUESTED** [review]
apart from those minor changes. It LGTM.

**Comment by nikul-serpentcs** [comment]
@max3903 @smangukiya Can you please update your review?

**Review by max3903 — APPROVED** [review]

**Pull request merged**